css/src/core/font-stack.scss

```diff
@@ -0,0 +1,25 @@
+html,
+body {
+	font-family: $normal-font-stack;
+	-webkit-font-smoothing: antialiased;
+	font-weight: $weight-normal;
+	text-rendering: optimizeLegibility;
+}
+
+kbd,
+pre,
+samp {
+	font-family: $monospace-font-stack !important;
+	-moz-osx-font-smoothing: auto;
+	-webkit-font-smoothing: auto;
+	line-height: 19px;
```

Comment:
testing required, but instead of a pix value, should we do a no-unit value? If we include `1.1875` does it have the same effect?

Reply:
It doesn't have the exact same effect because the unitless value is a factor of the element's font-size (currently 14px in docs-ui). The correct value would be ~1.3571 if we keep the current <pre> font size. Perhaps this line-height should be moved to wherever we'll declare the font-size for <pre> due to this relationship.

Reply:
@v-mingho agreed, moving it to the rule where we set the font size makes sense.

Reply:
@wibjorn I merged the PR for Docs-ui. Updated to 1.1875 here with the assumption that the pre will inherit a font size of 1rem/16 px. As there won't be any default element styles, I think this line height remain here for now.

Reply:
Not seeing any changes. Could you help me understand why the unitless value would need to be different in the two places? My understanding is the the unitless values change in reference to the base font-size. Wouldn't we want the height to scale between font size changes?

Reply:
Github isn't doing a good job at reflecting the latest changes. Click on the Files Changed tab for the most recent version.

The unitless value is multiplied by the element's own font size rather than the base font size, so it will scale upon font size changes of the <pre> element. It seems like Normalize.css sets the <pre> font size to 1em and <html> line height to 1.15. 

Without any default styling, then presumably the <pre> element will inherit the base font size of 16px (?). To set an unitless value that's equivalent to 19px, the factor will need to be 19/16 = 1.1875. 

In Docs-ui, we're setting the <pre> default font size to 0.875rem (14px) so the unitless value equivalent to 19px would be 19/14 = 1.3571, hence the two different values. 

If we pair a line height 1.3571 with 16px font size, it will look like this:
![image](https://user-images.githubusercontent.com/64044661/112535919-1008e400-8d6a-11eb-8075-92e0dfe60a24.png)

1.1871 with 16px font size:
![image](https://user-images.githubusercontent.com/64044661/112536531-d71d3f00-8d6a-11eb-96bb-784d1e28a066.png)

1.1871 with 14px:
![image](https://user-images.githubusercontent.com/64044661/112536960-5874d180-8d6b-11eb-9c0d-622546adedff.png)

1.3571 with 14px
![image](https://user-images.githubusercontent.com/64044661/112537058-780bfa00-8d6b-11eb-92d8-5de73802efc8.png)

Reply:
I compared a 16px font with line height 1.1871 vs the Normalize default 1.15. The visual difference is actually rather minimal. Shall we just remove the line height and take Normalize's default for now?

![image](https://user-images.githubusercontent.com/64044661/112538939-a5f23e00-8d6d-11eb-9e63-119a3232b6e1.png)

Reply:
Line height removed